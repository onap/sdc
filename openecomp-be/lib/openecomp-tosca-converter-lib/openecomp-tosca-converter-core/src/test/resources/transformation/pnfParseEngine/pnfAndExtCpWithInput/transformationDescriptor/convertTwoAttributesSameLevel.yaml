- transformation-for: nodeTemplate
  query:
    type: tosca.nodes.nfv.PnfExtCp
  conversions:
    - query:
        type:
      to-name: type
      to-value:
        strategy: replace
        from: tosca.nodes.nfv.PnfExtCp
        to: org.openecomp.resource.cp.v2.extCP
    - query:
        properties:
          layer_protocols:
      to-name: ip_requirements
      to-value:
        strategy: replaceInList
        list:
          - from: ipv4
            to:
              assingment_method: dhcp
              ip_version: 4
          - from: ipv6
            to:
              assingment_method: dhcp
              ip_version: 6
    - query:
        properties:
          description:
      to-name: description_converted
      to-value:
        strategy: copy