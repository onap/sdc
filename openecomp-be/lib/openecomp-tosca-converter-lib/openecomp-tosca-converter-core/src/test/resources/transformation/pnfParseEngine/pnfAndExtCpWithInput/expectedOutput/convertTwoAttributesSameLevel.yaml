topology_template:
  node_templates:
    pnfExtCp_1:
      type: org.openecomp.resource.cp.v2.extCP
      properties:
        ip_requirements:
          - assingment_method: dhcp
            ip_version: 4
          - assingment_method: dhcp
            ip_version: 6
        description_converted: External connection point to access this pnf